### ISSUE TITLE
Update command repeatedly prints out 2 same URLs (OSX)

### ISSUE THREAD

**Issue by muminoff** [minio/minio#2889]
## Expected behaviour

`minio update` result should not be 2 URLs
## Actual behaviour

`minio update` command returns 2 URLs
## Steps to reproduce the behaviour

``` bash
$ minio update

┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃  New update: https://dl.minio.io/server/minio/release/darwin-amd64/minio?update=yes ┃
┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛


┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃  New update: https://dl.minio.io/server/minio/release/darwin-amd64/minio?update=yes ┃
┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛


```
## Minio version

```
$ minio version

┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃  New update: https://dl.minio.io/server/minio/release/darwin-amd64/minio?update=yes ┃
┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛

Version: 2016-09-11T17:42:18Z
Release-Tag: RELEASE.2016-09-11T17-42-18Z
Commit-ID: 85e2d886bcb005d49f3876d6849a2b5a55e03cd3
```
## System information

OS: OSX El Capitan 10.11.6
Go: go1.7.1 darwin/amd64

**Comment by vadmeste** [comment]
@muminoff, can you paste the output of this command in your terminal:

`$ alias | grep minio`

**Comment by muminoff** [comment]
@vadmeste 

There is nothing aliased with `minio`:

``` bash
eyas:~ sardor$ alias |grep minio
eyas:~ sardor$
```

**Comment by vadmeste** [comment]
Thanks @muminoff.

@harshavardhana, this is simply a bug but reproducible only with release binary. There is an automatic fetch for new version for all minio commands, that's why update banner is showed twice with "minio update"

**Comment by harshavardhana** [comment]
> @harshavardhana, this is simply a bug but reproducible only with release binary. There is an automatic fetch for new version for all minio commands, that's why update banner is showed twice with "minio update"

@vadmeste ah i see we need to fix this.  You can easily validate this by removing the 'strings.HasPrefix' check.

**Comment by harshavardhana** [comment]
This will be fixed in pr #2892

**Comment by vadmeste** [comment]
> This will be fixed in pr #2892

I reviewed that PR @harshavardhana, I think one thing missing is that fetch update is done in cmd/main.go and in cmd/update-main.go, that's why update banner is showed twice.

**Comment by harshavardhana** [comment]
> I reviewed that PR @harshavardhana, I think one thing missing is that fetch update is done in cmd/main.go and in cmd/update-main.go, that's why update banner is showed twice.

Yeah its not finished yet. Fixing this last part.

**Comment by harshavardhana** [comment]
Fixed by #2892
